nexus/deployment/src/planner.rs

```diff
@@ -133,7 +133,30 @@ impl<'a> Planner<'a> {
                 }
             }
 
-            if ncrucibles_added > 0 {
+            // TODO XXX To test Nexus zone addition, attempt to ensure every
+            // sled has a Nexus zone. This is NOT a sensible policy, and should
+            // be removed before this lands on `main`.
+            //
+            // Adding a new Nexus currently requires the parent blueprint to
+            // have had at least one Nexus. By trying to ensure a Nexus on every
```

Comment:
I'm confused about why this paragraph and the next are here, but maybe it's obviated by the policy changes we talked about offline.